Dockerfile.tmpl

```diff
@@ -1,54 +1,39 @@
-FROM golang:1.14.0-alpine3.11 as builder
-
-WORKDIR /src
-
-RUN apk add --no-cache \
-    git \
-    ca-certificates
-
-ENV CADDY_SOURCE_VERSION={{ .config.caddy_version }}
-
-RUN git clone -b $CADDY_SOURCE_VERSION https://github.com/caddyserver/caddy.git --single-branch
-
-WORKDIR /src/caddy/cmd/caddy
-
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
-    go build -trimpath -tags netgo -ldflags '-extldflags "-static" -s -w' -o /usr/bin/caddy
-
-# Fetch the latest default welcome page and default Caddy config
-FROM alpine:3.11.3 AS fetch-assets
-
-RUN apk add --no-cache git
-
-ENV DIST_COMMIT=80870b227ded910971ecace4a0c136bf0ef46342
-
-WORKDIR /src/dist
-RUN git clone https://github.com/caddyserver/dist .
-RUN git checkout $DIST_COMMIT
-
-RUN cp config/Caddyfile /Caddyfile
-RUN cp welcome/index.html /index.html
-
-RUN mkdir -p /data/caddy
-RUN mkdir -p /config/caddy
-
 {{ include "base" | strings.TrimSpace }}
 
-COPY --from=builder /usr/bin/caddy /usr/bin/caddy
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs
-
-COPY --from=fetch-assets /Caddyfile /etc/caddy/Caddyfile
-COPY --from=fetch-assets /index.html /usr/share/caddy/index.html
-
-COPY --from=fetch-assets /data /data
-COPY --from=fetch-assets /config /config
+RUN apk add --no-cache ca-certificates
+
+# https://github.com/caddyserver/dist/commits
+ENV CADDY_DIST_COMMIT 80870b227ded910971ecace4a0c136bf0ef46342
```

Comment:
Should this be a config placeholder? Not a new change but figured I'd point it out. Maybe `{{ .config.dist_commit }}`

Reply:
yeah, good point - that can be templated later.

TBH I'm not 100% happy with the `index.html` (or the `Caddyfile` for that matter) that are in `dist`, they're not _quite_ right for use in a container... Maybe "someday" we can fix that 😉